railties/lib/rails/generators/rails/app/templates/config/databases/mysql.yml.tt

```diff
@@ -22,15 +22,25 @@ default: &default
 <% end -%>
 
 development:
-  <<: *default
-  database: <%= app_name %>_development
+  primary:<% unless options[:skip_solid_cache] %> &development_primary<% end %>
+    <<: *default
+    database: <%= app_name %>_development
+<% unless options[:skip_solid_cache] -%>
```

Comment:
I'm not positive if `skip_solid_cache?` is available here, but I think it is based on other database templates. If so, would be good to use it here too.